### ISSUE TITLE
Can't make or receive calls on Android

### ISSUE THREAD

**Issue by NIyerAD** [react-native-webrtc/react-native-callkeep#182]
Hi, I've been trying to get this package working on an Android project for some time now and I can't seem to make outbound calls or receive incoming calls even though it works completely fine on iOS. The `didReceiveStartCallAction` event fires on an outbound call which then continues to the native call UI screen but after a couple of seconds ends the call. On an incoming call the `didDisplayIncomingCall` event doesn't even fire. The only error I get in Android Studio's logs is `E/RNCK:VoiceConnectionService Constructor`.

Just to give a few more details, the physical device i'm using a Samsung Galaxy J4+ running on Android 9. The phone itself doesn't have a SIM card so i'm not sure if that could be the reason why it can't make or receive calls. 

Any help is appreciated.

Cheers

**Comment by stephanoparaskeva** [comment]
This is an issue for myself and I'm wondering why this has seen no attention?

**Comment by NIyerAD** [comment]
So I finally managed to solve this. Luckily [this guy](https://react-native-webrtc.discourse.group/t/issues-following-upgrade-to-1-75-1-solved/350) had the same issue. Turns out that Android doesn't necessarily like `advanced` audio constraints so if you're still having problems with this then check your audio constraints that you apply to `getUserMedia()`. If the constraints have 'advanced' options, strip it down to `audio: true` and see if that works.
